tests/tensor_search/test_add_documents.py

```diff
@@ -358,7 +358,7 @@ def test_add_documents_validation(self):
             add_res = tensor_search.add_documents(
                 config=self.config, add_docs_params=AddDocsParams(
                     index_name=self.index_name_1,
-                   docs=bad_doc_arg, auto_refresh=True, update_mode='update', device="cpu"
```

Comment:
Why was `update_mode='update'` here before? Maybe for previous tests that had that as the update mode, they could have the `use_existing_tensors` flag turned on